contents/blog/new-pricing.md

```diff
@@ -0,0 +1,143 @@
+  classes="rounded"
+/>
+
+Below you can see how our new pricing compares to Mixpanel's up to 20 million events – the limit of its "transparent" pricing. 
+
+<ProductScreenshot
+  imageLight={PricingComparison} 
+  imageDark={PricingComparisonDark} 
+  alt="New PostHog pricing compared to Mixpanel" 
+  classes="rounded"
+/>
+
+The table below is a side-by-side comparison of our new base event pricing, and pricing for events with person profiles enabled.
+
+Base events and person profiles combined cost the same as our previous per event pricing.
+
+| **Monthly events** | **New base event price** | **Base events + person profiles** | **Percentage diff.** |
+|--------------------|--------------------------|-----------------------------------|----------------|
+| **Up to 1m**       | Free                     | Free                              | -              |
+| **1-2m**           | $0.0000500 / event       | $0.0002480 / event                | -80%           |
+| **2-15m**          | $0.0000343 / event       | $0.0001040 / event                | -67%           |
+| **15-50m**         | $0.0000295 / event       | $0.0000655 / event                | -55%           |
+| **50-100m**        | $0.0000218 / event       | $0.0000364 / event                | -40%           |
+| **100-250**        | $0.0000150 / event       | $0.0000187 / event                | -20%           |
+| **∞**              | $0.0000090 / event       | $0.0000100 / event                | -10%           |
+
+## When should I use the new option?
+
+We use it for events where we're only interested in aggregate data, such as for logged-out users on our website.
+
+It could also be useful when:
+
+- You're willing to lose some functionality for lower bills.
+- You want to track API or server events, especially at high volumes. 
+- You run a large e-commerce or content website with millions of users.
+- You're not tracking a high-volume event because it would be prohibitively expensive.
+- You want to use [autocapture](/docs/product-analytics/autocapture) to track events, but have turned it off to save money.
+
+## How do I start using it?
+
+For most people, it only requires a simple config change to include `process_persons: "identified_only"`. 
+
+This will track anonymous traffic without person profiles, and start collecting person information any time an identifying action is taken (i.e. using `identify()`, `group()`, setting person properties with `$set`, etc).
+
+You can get instructions for this and other use cases in our [persons documentation](/docs/data/persons).
+
+## What's the downside?
+
+When sending events without person information, you cannot:
+
+- Filter on persons (e.g. an individual user)
+- Create cohorts filtered on person properties (e.g. paying vs non-paying users)
+- Target by person properties for feature flags, A/B tests and surveys.
+- Receive initial UTM values for tracking marketing campaign performance (only the most recent).
```

Comment:
Isn't there initial session UTM value?

Reply:
Afaik, no, but it is passed when an unidentified user becomes an identified user – e.g. they visit website and then signup.